go/interpreter/lfvm/interpreter.go

```diff
@@ -423,7 +427,7 @@ func steps(c *context, one_step_only bool) {
 		}
 
 		// Execute instruction
-		switch op {
+		switch op { // coverage-ignore opcodes are tested by CT
```

Comment:
yes, this can be done. I thought it would best to ignore this `switch` because it is such a large block of code, that we know is tested by another big project of this repo. But yes, an override can be done for this file anyway.